routes/auth.routes.js

```diff
@@ -0,0 +1,10 @@
+const router = require('express').Router()
+const {
+    register,
+    login
+} = require('../controller/auth.controller');
+
+router.post('/register', register);
+router.post('/login', login);
```

Comment:
dont understand

Reply:
Basically you have to ensure the number of requests a user can make at a time, this can be configured so that a user does not send way too many requests or it does not fall to a DDoS attack

Reply:
thank you I will fix